### ISSUE TITLE
Cannot find the specified map (E-D3). Could this be fixed? If so what needs to be done?

### ISSUE THREAD

**Issue by ghost** [Egoistically/ALAuto#132]
When i enter E-D3 as map to repeat. It shows me the message that it can't find the map. Does this have to do with the smoke coming in front of the image?

**Comment by hjk22** [comment]
That's right. Remaking the asset for said map will solve the problem. I suggest you use the part less affected by the smoke.
In the [wiki](https://github.com/Egoistically/ALAuto/wiki/Creating-new-assets-for-bot) you can find a guide to make your own asset.

**Comment by David1544** [comment]
Anyone have any luck with this?
Unfortunately since the text box is slightly transparent all my attempts at using that has failed, and screencapping the "clear" part fails as well: https://i.imgur.com/vHVenJv.png

Edit: I realize this issue was probably posted in regards to the Crimson Echoes re-run, but this issue is present in the Iris of Light and Dark rerun as well.

**Comment by David1544** [comment]
Update: Referred to issue #137 , adjusting the similarity value down to 0.96 made the issue go away.

**Comment by empathist-dot** [comment]
@David1544  Did the bot work on D3 aftewards though? adjusting the similarity value made it able to find the stage, but it can't farm it.

![unknown](https://user-images.githubusercontent.com/61105263/83323454-297ef380-a25f-11ea-95a9-4cbefb674233.png)
